pkg/v1/tkg/test/tkgctl/shared/package.go

```diff
@@ -125,26 +124,42 @@ func getClients(ctx context.Context, exportFile string) (k8sClient client.Client
 
 // getPackagesFromCB gets the list of packages from CB that are supposed to be installed on a cluster
 func getPackagesFromCB(ctx context.Context, clusterBootstrap *runtanzuv1alpha3.ClusterBootstrap, mccl, wccl client.Client, mcClusterName, mcClusterNamespace, wcClusterName, wcClusterNamespace, infrastructureName string, isManagementCluster bool) ([]kapppkgv1alpha1.Package, error) {
-	var packages []kapppkgv1alpha1.Package
+	var (
+		packages        []kapppkgv1alpha1.Package
+		systemNamespace string
+	)
+
+	if infrastructureName == "tkgs" {
+		systemNamespace = constants.TKGSClusterClassNamespace
```

Comment:
@chandrareddyp , As an example,  I have modified functions to use "vmware-system-tkg"  as these packages exist in that namespace
cc: @shyaamsn